buildimage.py

```diff
@@ -454,7 +454,15 @@ def do_build(self, plugin, variant, prod):
         super().do_build(plugin, variant, prod)
 
         # Copy rootfs_overlay first
-        cmd = "cp -rf profile/" + self.profiletype + "/rootfs_overlay" + " " + self.imgdir
+        #cmd = "cp -rf profile/" + self.profiletype + "/rootfs_overlay" + " " + self.imgdir
+        #run_cmd(cmd)
+
+        #Rename variant.apk as app.apk
+        cmd = "cp rootfs_overlay/var/www/html/public/" + variant + ".apk" + " " + "rootfs_overlay/var/www/html/public/app.apk"
```

Comment:
Move this to the plugin repository

Reply:
Along with the `app.apk` for each variant and update the `initialize_plugin.sh` script to copy the required variant app to `rootfs_overlay/.../public` folder